scipy/cluster/hierarchy.py

```diff
@@ -1674,15 +1674,15 @@ def _plot_dendrogram(icoords, dcoords, ivl, p, n, mh, orientation,
 
         lbls = ax.get_xticklabels()
         if leaf_rotation:
-            map(lambda lbl: lbl.set_rotation(leaf_rotation), lbls)
+            list(map(lambda lbl: lbl.set_rotation(leaf_rotation), lbls))
```

Comment:
Why not get rid of the UUOM(*) and just write loops?

```py
for lbl in lbls:
    lbl.set_rotation(leaf_rotation)
```

This is much more readable.

(*)  Useless use of `map`